l10n_fr_ecotaxe_sale/models/sale.py

```diff
@@ -0,0 +1,46 @@
+# © 2015 -2020 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+
+    _inherit = "sale.order.line"
+
+    subtotal_ecotaxe = fields.Float(
+        store=True, compute="_compute_ecotaxe", oldname="amount_ecotaxe"
```

Comment:
```suggestion
        store=True, compute="_compute_ecotaxe"
```

Otherwise getting these warnings:
```
WARNING testdb odoo.fields: Field sale.order.line.subtotal_ecotaxe: unknown parameter 'oldname', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it 
```

`oldname` has been deprecated many versions ago, and in any case it should be removed during a migration IMO